tusclient/uploader/baseuploader.py

```diff
@@ -54,6 +54,9 @@ class BaseUploader:
         - retry_delay (int):
             How long (in seconds) the uploader should wait before retrying a failed upload attempt.
             If not specified, it defaults to 30.
+        - verify (bool):
```

Comment:
The parameter should have a more descriptive name, e.g. verify_tls_certificate. This way it is clear that it is about TLS and no some kind of upload verification.

Reply:
Renamed the parameter to `verify_tls_cert`.